solr/packaging/test/test_export.bats

```diff
@@ -29,7 +29,7 @@ teardown() {
 }
 
 @test "Check export command" {
-  run solr start -c -Dsolr.modules=sql
```

Comment:
Ummm why is this removed? This is checking a query so need sql module right?

Reply:
I don't think it uses the sql module..  it's just a straight up solr query, not a sql one....